test/htmltojsx-test.js

```diff
@@ -136,6 +136,42 @@ describe('htmltojsx', function() {
       expect(converter.convert('<div style="font-size: 12pt">Test</div>').trim())
         .toBe('<div style={{fontSize: \'12pt\'}}>Test</div>');
     });
+	
+	it('should convert vendor-prefix "style" attributes', function() {
```

Comment:
Thanks! I'll have to see if I can figure out which linting tools to use, as I don't have experience with setting up linters.